script/deploy/healthCheck.ts

```diff
@@ -239,278 +239,356 @@ const main = defineCommand({
+            }
+        }
 
-      if (sigsToApprove.length > 0)
-        logError(`Missing ${sigsToApprove.length} DEX signatures`)
-      else consola.success('No missing signatures.')
+        if (sigsToApprove.length > 0)
+          logError(`Missing ${sigsToApprove.length} signatures`)
+        else consola.success('No missing signatures.')
 
-      //          ╭─────────────────────────────────────────────────────────╮
-      //          │                Check contract ownership                 │
-      //          ╰─────────────────────────────────────────────────────────╯
-      consola.box('Checking ownership...')
+        //          ╭─────────────────────────────────────────────────────────╮
+        //          │                Check contract ownership                 │
+        //          ╰─────────────────────────────────────────────────────────╯
+        consola.box('Checking ownership...')
 
-      const refundWallet = getAddress(globalConfig.refundWallet)
-      const feeCollectorOwner = getAddress(globalConfig.feeCollectorOwner)
+        const refundWallet = getAddress(globalConfig.refundWallet)
+        const feeCollectorOwner = getAddress(globalConfig.feeCollectorOwner)
 
-      // Check ERC20Proxy ownership
-      const erc20ProxyOwner = await erc20Proxy.read.owner()
-      if (getAddress(erc20ProxyOwner) !== getAddress(deployerWallet))
-        logError(
-          `ERC20Proxy owner is ${getAddress(
-            erc20ProxyOwner
-          )}, expected ${getAddress(deployerWallet)}`
-        )
-      else consola.success('ERC20Proxy owner is correct')
+        // Check ERC20Proxy ownership
+        const erc20ProxyOwner = await erc20Proxy.read.owner()
+        if (getAddress(erc20ProxyOwner) !== getAddress(deployerWallet))
+          logError(
+            `ERC20Proxy owner is ${getAddress(
+              erc20ProxyOwner
+            )}, expected ${getAddress(deployerWallet)}`
+          )
+        else consola.success('ERC20Proxy owner is correct')
+
+        // Check that Diamond is owned by SAFE
+        if (networksConfig[network.toLowerCase()].safeAddress) {
+          const safeAddress = networksConfig[network.toLowerCase()].safeAddress
 
-      // Check that Diamond is owned by SAFE
-      if (networksConfig[network.toLowerCase()].safeAddress) {
-        const safeAddress = networksConfig[network.toLowerCase()].safeAddress
+          await checkOwnership(
+            'LiFiDiamond',
+            safeAddress,
+            deployedContracts,
+            publicClient
+          )
+        }
 
+        // FeeCollector
         await checkOwnership(
-          'LiFiDiamond',
-          safeAddress,
+          'FeeCollector',
+          feeCollectorOwner,
           deployedContracts,
           publicClient
         )
-      }
 
-      // FeeCollector
-      await checkOwnership(
-        'FeeCollector',
-        feeCollectorOwner,
-        deployedContracts,
-        publicClient
-      )
+        // Receiver
```

Comment:
We dont deploy "Receiver" contracts anymore. 
Is this failing silently or why have we not noticed this yet?

Reply:
ok